src/components/dialogs/edit-package-details-dialog.tsx

```diff
@@ -150,13 +150,13 @@ export const EditPackageDetailsDialog = ({
       if (hasLicenseChanged) {
         if (packageFiles.includes("LICENSE") && !licenseContent) {
           await axios.post("/package_files/delete", {
-            package_name_with_version: `${packageAuthor}/${formData.unscopedPackageName}`,
+            package_name_with_version: `${packageAuthor}/${formData.unscopedPackageName}@latest`,
```

Comment:
I don't think we need to specifically add this, the backend fetches the latest on default case

```suggestion
            package_name_with_version: `${packageAuthor}/${formData.unscopedPackageName}`,
```